packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -702,19 +663,27 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
     /**
      * @notice Updates the voter's trackers for staking and voting, specifying a maximum number of resolved requests to
      * traverse. This function can be used in place of updateTrackers to process the trackers in batches, hence avoiding
-     * potential issues if the number of elements to be processed is big and the associated gas cost is too high.
+     * potential issues if the number of elements to be processed is large and the associated gas cost is too high.
      * @param voterAddress address of the voter to update the trackers for.
-     * @param maxTraversals last price request index to update the trackers for.
+     * @param maxTraversals maximum number of resolved requests to traverse in this call.
      */
     function updateTrackersRange(address voterAddress, uint64 maxTraversals) external {
         resolveResolvablePriceRequests();
         _updateAccountSlashingTrackers(voterAddress, maxTraversals);
     }
 
+    /**
+     * @notice Resolves all resolvable price requests. This function traverses all pending price requests and resolves
+     * them if they are resolvable. It also rolls requests, if needed and deletes requests, if required.
+     */
     function resolveResolvablePriceRequests() public {
```

Comment:
Should we name this function differently?

`processResolvablePriceRequests`?

Reply:
sure. done